app/templates/components/nav-bar.hbs

```diff
@@ -48,7 +48,7 @@
       <i class="large content icon"></i>
     </a>
     {{#link-to 'index' class='item' activeClass=''}}
-      <div class="ui header small text">{{config.appName}}</div>
+      <div class="ui header small text">eventyay</div>
```

Comment:
Revert this and change eventyay to `{{config.appName}}` elsewhere

Reply:
> 
> 
> Revert this and change eventyay to `{{config.appName}}` wherever found in the project
Okk i will work on this.
But since we always have to show eventyay there, i think that there is nothing wrong in directly writing eventyay.

Reply:
> since we always have to show eventyay there

Who said that?

Reply:
This is not eventyay repository, this is open-event-frontend, eventyay is an instance of open-event.

Same difference as Chromium vs Chrome.

Reply:
okk

> 
> 
> This is not eventyay repository, this is open-event-frontend, eventyay is an instance of open-event.
> 
> Same difference as Chromium vs Chrome.

okk

Reply:
> > Revert this and change eventyay to `{{config.appName}}` wherever found in the project
> 
> Okk i will work on this.
> But since we always have to show eventyay there, i think that there is nothing wrong in directly writing eventyay.

We want to show app name here which is entered in admin panel.
Make it `settings.appName`

Reply:
> We want to show app name here which is entered in admin panel.
> Make it `settings.appName`

@kushthedude changed